mnemosyne/core/banks.py

```
@@ -25,7 +25,7 @@
 import shutil
 import sqlite3
 from pathlib import Path
-from typing import List, Optional
+from typing import List
```

Ruff (0.15.21)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)